app/models/workflow_step.rb

```diff
@@ -2,7 +2,7 @@
 
 # Models a process that occurred for a digital object. Basically a log entry.
 class WorkflowStep < ApplicationRecord
-  validates :druid, presence: true
+  validates :druid, format: { with: /\Adruid:[a-zA-Z]{2}\d{3}[a-zA-Z]{2}\d{4}/, message: 'is invalid format or missing prefix' }
```

Comment:
Open to using the druidtools validator here? https://github.com/sul-dlss/druid-tools/blob/90be3a6fbeffe0be2822dd4ba9555ec80e4b6e12/README.md#check-whether-a-druid-is-valid

Reply:
yes, that's fine, though will still need to check for a druid prefix, which DruidTools does not do